rocAL/rocAL/source/pipeline/master_graph.cpp

```diff
@@ -796,13 +796,78 @@ MasterGraph::copy_out_tensor(void *out_ptr, RocalTensorFormat format, float mult
                     else if(output_data_type == RocalTensorDataType::FP16) {
                         half *output_tensor_16 = static_cast<half *>(out_ptr);
                         auto channel_size = w * h;
+    #if (ENABLE_SIMD && __AVX2__)
+                        half *B_buf = output_tensor_16 + dest_buf_offset;
```

Comment:
Rename B_buf, G_buf, R_buf as B_buf_16, G_buf_16, R_buf_16

Reply:
Resolved in latest commit